examples/mistral/config.yml

```diff
@@ -0,0 +1,69 @@
+base_model: mistralai/Mistral-7B-v0.1
+base_model_config: mistralai/Mistral-7B-v0.1
+model_type: MistralForCausalLM
+tokenizer_type: LlamaTokenizer
+
+load_in_8bit: false
+load_in_4bit: false
+strict: false
+
+datasets:
+  - path: mhenrichsen/alpaca_2k_test
+    type: alpaca
+dataset_prepared_path: last_run_prepared
+val_set_size: 0.01
+output_dir: ./out
+
+sequence_len: 4096
```

Comment:
Isn't mistral 8k?

Reply:
Thanks! As of that time, there was no flash, so I just thought to use a reasonable balanced number. However, I will now update it now that we have FA patch.